pico_project.py

```diff
@@ -796,6 +796,7 @@ def CheckSDKPath(gui):
 def ParseCommandLine():
     parser = argparse.ArgumentParser(description='Pico Project generator')
     parser.add_argument("name", nargs="?", help="Name of the project")
+    parser.add_argument("-t", "--tsv", help="Set an alternative pico_configs.tsv file", default="pico_configs.tsv")
```

Comment:
I think I'd word this as "Select an ..." rather than "Set an ..."

Reply:
I think I messed up my commit history when trying to keep my work up to date with the upstream. I have made the required changes, though. What do you suggest I do?